sphinx_doc/user_guide.rst

```diff
@@ -98,9 +100,14 @@ An admin user can add or remove data from the genomics-workspace database via th
 
 * ``python manage.py blast_shown [path of fasta file] -shown ‘true’`` (e.g python manage.py blast_shown media/blast/db/GCF_003254395.2_Amel_HAv3.1_genomic.fna -shown ‘true’)
 
+If you don't see the new organism under the blast webapp, you can login into the postgres database and see if the ``blast_blastdb`` table has the rows with information about your organism.
 
+6. To show the genome in jbrowse
 
-HMMER Database Configuration
+* ``python manage.py addjbrowse [genome assembly fasta file] -shown ‘true’`` (e.g python manage.py addjbrowse GCF_003254395.2_Amel_HAv3.1_genomic.fna https://apollo.nal.usda.gov/apollo/[genus]%20[species]/jbrowse/)
```

Comment:
Change to: * ``python manage.py addjbrowse [genome assembly fasta file] [Jbrowse URL]`` (e.g python manage.py addjbrowse GCF_003254395.2_Amel_HAv3.1_genomic.fna https://apollo.nal.usda.gov/apollo/[genus]_[species]/jbrowse/)

Reply:
Oops. Copy paste vestige. Done